cmd/statusd/utils.go

```diff
@@ -133,15 +133,15 @@ func testVerifyAccountPassword(t *testing.T) bool {
 	}
 	defer os.RemoveAll(tmpDir) // nolint: errcheck
 
-	if err = common.ImportTestAccount(tmpDir, "test-account1.pk"); err != nil {
+	if err = common.ImportTestAccount(tmpDir, "test-account3.pk"); err != nil {
```

Comment:
Why so many test accounts? There were only 2, let's leave just them?

Reply:
It has been done to not to rewrite tests with old account1 and new auth (BIP-some-number).

Reply:
Can you elaborate on that?